spec/models/cat_spec.rb

```diff
@@ -1,5 +1,33 @@
 require 'rails_helper'
 
+# RSpec.describe Cat, type: :model do
+#   pending "add some examples to (or delete) #{__FILE__}"
+# end
+
 RSpec.describe Cat, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it "name should not be empty" do
+    cat = Cat.create age: 5, enjoys: 'Furrrrociously hunting bugs.'
+    expect(cat.errors[:name]).to_not be_empty
+  end
 end
+
+RSpec.describe Cat, type: :model do
```

Comment:
You can have multiple describe statements but you don't need to. All the additional tests can be `it` statements wrapped inside the one describe block.